astropy/wcs/__init__.py

```diff
@@ -1,5 +1,6 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
-"""
+"""World Coordinate System (WCS) transformations in FITS files.
```

Comment:
Interesting... Where did this line come from?

Reply:
For a few it was necessary to add the summary line, which was missing (https://numpydoc.readthedocs.io/en/latest/format.html#short-summary).
This occurred in a few locations of very old docs, I think before the adoption of numpydoc as our standard. This line for example was last edited in 2011.